lib/phoenix/presence.ex

```diff
@@ -137,9 +137,9 @@ defmodule Phoenix.Presence do
 
   """
 
-  @type presences :: %{String.t => %{metas: [map()]}}
-  @type presence :: %{key: String.t, meta: map()}
-  @type topic :: String.t
+  @type presences :: %{String.t() => %{metas: [map()]}}
```

Comment:
All the changes until the end of the file was `mix format`, I'm assuming we want to run that everywhere but I put it in a separate commit in case we don't.